src/LayoutManager/ActiveSelectionLayoutManager.ts

```diff
@@ -1,27 +1,57 @@
-import { LayoutManager, buildStandardEvents } from './LayoutManager';
+import { LayoutManager } from './LayoutManager';
 import type { RegistrationContext, StrictLayoutContext } from './types';
 import type { FabricObject } from '../shapes/Object/FabricObject';
 
 /**
- * The ActiveSelectionLayoutManager is exactly as the LayoutManager
- * But what it does differently is that it subscribes the activeSelection
- * modification events to modify the layout of the original group of the
- * children. This is done by swapping target and object in the subscribeTargets
- * method. All the rest is identical
+ * Today the LayoutManager class also takes care of subscribing event handlers
+ * to update the group layout when the group is interactive and a transform is applied
+ * to a child object.
+ * The ActiveSelection is never interactive, but it could contain obejcts from
+ * groups that are.
+ * The standard LayoutManager would subscribe the children of the activeSelection to
+ * perform layout changes to the activeselection itself, what we need instead is that
+ * the transformation applied to the active selection will trigger changes to the
+ * original group of the children ( the one referenced under the parent property )
+ * This subclass of the LayoutManager has a single duty to fill the gap of this difference.
  */
 export class ActiveSelectionLayoutManager extends LayoutManager {
   /**
-   * subscribe to object layout triggers
+   * Subscribe an object to transforms events that will trigger a layout change in the group
+   * This is important only for interactive groups.
+   * @param object
+   * @param context
    */
-  protected attachHandlers(
-    childObject: FabricObject,
+  protected subscribe(
+    object: FabricObject,
     context: RegistrationContext & Partial<StrictLayoutContext>
-  ): (() => void)[] {
-    const { parent, group: activeSelection } = childObject;
+  ) {
+    const { parent, group: activeSelection } = object;
     if (!parent || !activeSelection || activeSelection === parent) {
       // nothing to do here
-      return [];
+      return;
+    }
+    this.unsubscribe(object, context);
+    const disposers = this.attachHandlers(activeSelection, {
+      target: parent,
+      targets: [object],
```

Comment:
I don't like this part.
It means the context of the trigger is incomplete. There might be more objects from this parent

Reply:
Looking at `attachHandlers` I don't see it requires `targets`
It used to but you changed it, right?
I do think it is good to have all the context available for all calls but it must be the correct one. Better none than a wrong one

Reply:
i kept it there because you asked for the context to not be removed.